features/items/items.feature

```diff
@@ -133,7 +133,48 @@ Feature: Amazon DynamoDB Items
       """
       {
         "id": "4",
-        "count": 5,
+        "rk": 5,
         "body": "Updated Body Content."
       }
       """
+
+  Scenario:  Increment Atomic Counter Attribute with a Custom Value
+    Given an aws-record model with data:
+      """
+      [
+        { "method": "string_attr", "name": "id", "hash_key": true },
+        { "method": "integer_attr", "name": "rk", "range_key": true },
+        { "method": "atomic_counter", "name": "counter" }
+      ]
+      """
+    And an item exists in the DynamoDB table with item data:
+      """
+      {
+        "id": "5",
+        "rk": 6,
+        "counter": 0
+      }
+      """
+    And we call the 'find' class method with parameter data:
```

Comment:
Is the 'find' here necessary?

Reply:
Yes, the `find` method is needed to use the `@instance` variable. I combed through the existing step definitions and couldn't find any steps that does both saving and finding the data.